### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by grentlich**

### CODE REVIEW

File: micro-app/ServiceNow ticket list/README.md [hunk 1]
```diff
@@ -8,15 +8,21 @@
 
 - Create a new micro-app according to [Declare your micro-app extension](https://developer.lumapps.com/portal/documentation/micro-app-steps/declare-your-microapp-extension.md). You can use the following icon url: `https://upload.wikimedia.org/wikipedia/commons/thumb/5/57/ServiceNow_logo.svg/220px-ServiceNow_logo.svg.png`.
 
-## Step 1: Get the list of tickets - API Call
+## Step 1: Get the end user s
```
Comment (gregorycat) [inline micro-app/ServiceNow ticket list/README.md:11]:
```suggestion
## Step 1: Get the end users
```

File: micro-app/ServiceNow ticket list/README.md [hunk 2]
```diff
@@ -8,15 +8,21 @@
...
 
-Create an API Call named `ticket_list_api` according to the following template: [ServiceNow: ticket_list_api](https://documenter.getpostman.com/view/32450244/2sA3kPq5Do#a9685d68-29f0-40b6-b8d6-975683cba7e9)
+Create an API Call named `getuser` according to the following endpoint as a GET Method:
+/api/now/table/sys_user?sysparm_query=email=${user.email}
+
+## Step 2: Get the list of tickets - API Call
+
+Create an API Call named `ticket_list_api` using the following endpoint as a GET Method: 
+/api/now/table/incident?sysparm_limit=5&sysparm_query=caller_id=${actions.getuser.response.body.result[0].sys_id}^stateNOT%20IN7
```
Comment (gregorycat) [inline micro-app/ServiceNow ticket list/README.md:19]: Are you sure of this URL ? the `^` is uncommon

**Review by gregorycat — CHANGES_REQUESTED** [review]

File: micro-app/ServiceNow ticket list/README.md (hunk 1 above)
Comment (Georgina-gh) [inline micro-app/ServiceNow ticket list/README.md:11]:
```suggestion
## Step 1: Get the end user's email
```

File: micro-app/ServiceNow ticket list/README.md [hunk 3]
```diff
@@ -8,15 +8,21 @@
 
 - Create a new micro-app according to [Declare your micro-app extension](https://developer.lumapps.com/portal/documentation/micro-app-steps/declare-your-microapp-extension.md). You can use the following icon url: `https://upload.wikimedia.org/wikipedia/commons/thumb/5/57/ServiceNow_logo.svg/220px-ServiceNow_logo.svg.png`.
 
-## Step 1: Get the list of tickets - API Call
+## Step 1: Get the end user s
 
-Create an API Call named `ticket_list_api` according to the following template: [ServiceNow: ticket_list_api](https://documenter.getpostman.com/view/32450244/2sA3kPq5Do#a9685d68-29f0-40b6-b8d6-975683cba7e9)
+Create an API Call named `getuser` according to the following endpoint as a GET Method:
```
Comment (Georgina-gh) [inline micro-app/ServiceNow ticket list/README.md:13]:
```suggestion
Create an API Call named `getuser` using the following endpoint, with the GET Method:
```

File: micro-app/ServiceNow ticket list/README.md [hunk 4]
```diff
@@ -8,15 +8,21 @@
...
+/api/now/table/sys_user?sysparm_query=email=${user.email}
```
Comment (Georgina-gh) [inline micro-app/ServiceNow ticket list/README.md:14]:
```suggestion
`/api/now/table/sys_user?sysparm_query=email=${user.email}`
```

File: micro-app/ServiceNow ticket list/README.md (hunk 2 above)
Comment (Georgina-gh) [inline micro-app/ServiceNow ticket list/README.md:19]:
```suggestion
`/api/now/table/incident?sysparm_limit=5&sysparm_query=caller_id=${actions.getuser.response.body.result[0].sys_id}^stateNOT%20IN7`
```

**Review by Georgina-gh — APPROVED** [review]
I made some suggestions because there is a typo in the step 1 title and so that hopefully the endpoints can have the code styling applied (`..`)

**Comment by grentlich** [comment]
ok go

File: micro-app/ServiceNow ticket list/README.md (hunk 2 above)
Comment (Georgina-gh) [reply micro-app/ServiceNow ticket list/README.md:19]: @grentlich check if you made an error in the endpoint:
/api/now/table/incident?sysparm_limit=5&sysparm_query=caller_id=${actions.getuser.response.body.result[0].sys_id}^stateNOT%20IN7

If you are sure the endpoint is tested and correct you can add a comment to confirm this.

File: micro-app/ServiceNow ticket list/README.md (hunk 1 above)
Comment (Georgina-gh) [reply micro-app/ServiceNow ticket list/README.md:11]: @grentlich commit my or Gregory's suggestion (click "commit suggestion") or make your own changes to fix the typo

File: micro-app/ServiceNow ticket list/README.md (hunk 2 above)
Comment (grentlich) [reply micro-app/ServiceNow ticket list/README.md]: oui

**Review by gregorycat — APPROVED** [review]

**Pull request merged**